docs/integrations/aws.md

```diff
@@ -419,9 +419,23 @@ If for any reason you have to use S3A, here are the instructions:
 To ensure integrity of uploaded objects, checksum validations for S3 writes can be turned on by setting catalog property `s3.checksum-enabled` to `true`. 
 This is turned off by default.
 
+### S3 Delete 
+
+When the catalog property `s3.delete-enabled` is set to `false`, the objects are not deleted from S3. 
+This is set to `true` by default.
```

Comment:
we should link to the next section with something in the line of "This is expected to be used in combination with s3 delete tagging", so objects are tagged and removed using S3 lifecycle policy. See xxx for more details".

Reply:
Actually, given that the delete-enabled is only relevant under the context of tagging, would it make more sense to just mention this in the tagging section below, instead of having its own section?